esp/src/eclwatch/WUDetailsWidget.js

```diff
@@ -156,18 +156,17 @@ define([
 
             _onCancelDialog: function () {
                 this.zapDialog.hide();
-                this.emailCheckbox.set("value", "false");
-                this.emailSubject.reset();
-                this.emailBody.reset();
                 this.checkThorLogStatus();
             },
 
             _onSubmitDialog: function(){
                 var includeSlaveLogsCheckbox = this.includeSlaveLogsCheckbox.get("checked");
+                var emailCheckbox = this.emailCheckbox.get("checked");
                 if (this.zapForm.validate()) {
                     //WUCreateAndDownloadZAPInfo is not a webservice so relying on form to submit.
                     //Server treats "on" and '' as the same thing.
                     this.includeSlaveLogsCheckbox.set("value", includeSlaveLogsCheckbox ? "on" : "off");
+                    this.emailCheckbox.set("value", emailCheckbox ? "on" : "off");
```

Comment:
I don't think this is needed as "SendMail" is a regular Boolean on the server (and am concerned "off" would be considered "true"?)